src/documentation/pages/components/tooltip/examples/index.vue

```diff
@@ -0,0 +1,31 @@
+<template lang="pug">
+  div
+    h2 Examples
+
+    <BaseTooltip />
+    div(v-if="['lumen', 'centurylink'].includes($store.state.themes.theme)")
+      <LightTooltip />
+    <DisabledTooltip />
+    <PositionedTooltip />
```

Comment:
Instead of naming them BaseTooltip, LightTooltip etc we could just name them Base, Light, Disabled, Positioning, etc